client/shipping-label/components/label-purchase-modal/packages-step/add-item.js

```diff
@@ -29,7 +30,7 @@ const AddItemDialog = ( {
 
 	const packageLabels = getPackageDescriptions( selected, all, true );
 	const getPackageNameElement = ( pckgId ) => {
-		return <span className="wcc-move-item-dialog__package-name">{ packageLabels[ pckgId ] }</span>;
+		return <span className="packages-step__dialog-package-option">{ packageLabels[ pckgId ] }</span>;
```

Comment:
Yep, fixed in https://github.com/Automattic/woocommerce-services/pull/1051/commits/c147468c7842f5ff171592b985e70e5fa674fcf7